static/js/src/release-chart.js

```diff
@@ -144,6 +155,9 @@ function clearCharts() {
   if (document.querySelector("#server-desktop-eol")) {
     document.querySelector("#server-desktop-eol").innerHTML = "";
   }
+  if (document.querySelector("#eol-1604")) {
+    document.querySelector("#eol-1604").innerHTML = "";
+  }
```

Comment:
This PR didn't introduce this pattern, but there are a lot of examples of querying the document for an ID then, if an element with that is found, querying the document again (unnecessarily) for the same thing. It'd be better to assign the query to a variable so the query is cached, and the document is only queried once (same with other examples in this file):

```suggestion
  const eol1604 = document.querySelector("#eol-1604");
  if (eol1604) {
    eol1604.innerHTML = "";
  }
```

Reply:
Makes sense, I'll update the rest in the file too